docs/download.md

```diff
@@ -52,27 +52,27 @@ do not have CUDA forward compatibility (for example, GeForce), CUDA 11.5 or late
 need to ensure the minimum driver (450.80.02) and CUDA toolkit are installed on each Spark node.
 
 ### Verify signature
-* Download the [RAPIDS Accelerator for Apache Spark 23.02.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.02.0/rapids-4-spark_2.12-23.02.0.jar)
-  and [RAPIDS Accelerator for Apache Spark 23.02.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.02.0/rapids-4-spark_2.12-23.02.0.jar.asc)
+* Download the [RAPIDS Accelerator for Apache Spark 23.04.0 jar](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.04.0/rapids-4-spark_2.12-23.04.0.jar)
+  and [RAPIDS Accelerator for Apache Spark 23.04.0 jars.asc](https://repo1.maven.org/maven2/com/nvidia/rapids-4-spark_2.12/23.04.0/rapids-4-spark_2.12-23.04.0.jar.asc)
 * Download the [PUB_KEY](https://keys.openpgp.org/search?q=sw-spark@nvidia.com).
 * Import the public key: `gpg --import PUB_KEY`
-* Verify the signature: `gpg --verify rapids-4-spark_2.12-23.02.0.jar.asc rapids-4-spark_2.12-23.02.0.jar`
+* Verify the signature: `gpg --verify rapids-4-spark_2.12-23.04.0.jar.asc rapids-4-spark_2.12-23.04.0.jar`
 
 The output if signature verify:
 
 	gpg: Good signature from "NVIDIA Spark (For the signature of spark-rapids release jars) <sw-spark@nvidia.com>"
 
 ### Release Notes
 New functionality and performance improvements for this release include:
-* Delta Lake MERGE/DELETE/UPDATE (experimental feature, can be enabled with a config flag)
-* Function `from_json`
-* Hive text table write 
-* Databricks 11.3 ML LTS support
-* Support batched full join to improve full join's performance
+* Introduces OOM retry framework for automatic OOM handling in memory-intensive operators, such as: join, aggregates and windows, coalescing, projections and filters.
+* Support dynamic repartitioning in large/skewed hash joins
+* Optimize the transpilation in `regexp_extract` function
+* Support Delta Lake write with auto-optimization and auto-compaction on Databricks platforms
 * Qualification and Profiling tool:
-  * EMR user tools support for qualification
-  * EMR user tools support for bootstrap
-  * Updated estimated speedup factors for on-prem, Dataproc, and EMR environments for qualification
+  * Add support to recommend cluster shape options on CSPs
+  * Add support for Databricks local mode with cost savings based on cluster metadata
+  * Add TCO calculator to estimate annualized cost savings, including estimated frequency for applications
+  * Add support in the qualification tool to generate estimated speed-up for ML functionality in Spark applications
```

Comment:
@mattahrens can you please review the Qual tool new features and make sure this looks good to you?

Reply:
Minor change: `Add support to recommend cluster shape options on CSPs` to `Add support to recommend cluster shape options on Dataproc and EMR`

Reply:
updated